Sources/hostmgr/helpers/S3.swift

```diff
@@ -125,7 +125,7 @@ struct S3Manager {
         for bucket: String,
         in region: Region
     ) throws -> S3 {
-        let timeout = TimeAmount.minutes(30)
+        let timeout = TimeAmount.hours(2)
```

Comment:
Nitpick. Since this timeout is huge, I think it would be useful to have an explanatory comment for it. What do you think?

Reply:
Good call. I think I'll tweak this value after I've done deploying the beta 6 image. I'll add some comments then.

Reply:
Sounds good.